game/scripts/npc/items/item_magic_wand.txt

```diff
@@ -37,10 +37,10 @@
 		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_IMMEDIATE | DOTA_ABILITY_BEHAVIOR_NO_TARGET"
     	"AbilityTextureName"            "custom/magic_wand_1"
 		"Model"							"models/props_gameplay/magic_wand.vmdl"
-		
+
 		// Stats
 		//-------------------------------------------------------------------------------------------------------------
-		"AbilityCooldown"				"13.0 12.0 11.0 10.0 9.0"
+		"AbilityCooldown"				"13.0 10.0 7.0 4.0 1.0"
```

Comment:
Might want to increase the last level a bit, to avoid double cast by accident. Like they did with Diffusal in Vanilla... though I guess double using it has little consequence.